python/testSrc/com/jetbrains/python/refactoring/classes/extractSuperclass/PyExtractSuperclassTest.java

```diff
@@ -134,6 +135,67 @@ public void testAbstractMethodDocStringIndentationPreserved() {
     });
   }
 
+  // PY-44858
+  public void testExtractNotCreateInitInAnotherDir(){
```

Comment:
These tests don't test anything. If you revert all the changes from `PyExtractSuperclassHelper`, they still pass. The reason is that they assert not the structure of the *actual* "Extract superclass" result, but the structure of their own *expected* result in `doTestNotCreateInitCommon`. 

For instance, in each of these tests the file where a superclass is expected to be placed is called "suppa.py" according to the test data, but it's not where the test actually puts it. It's always an `__init__.py` file inside a new package, e.g. in this test the superclass ends up in in `mypkg/b/__init__.py`, but you check the structure of the directories `a` and `b` copied from the test data verbatim. Please add tests that behave like scenarios in PyMoveTest, comparing the project directory before and after the refactoring.

Reply:
After the refactoring is performed the test project directory looks like this (with the bug fix code reverted):
```
mypkg/
  b/
    __init__.py # Suppa is here
  __init__.py
a/
  source.py
b/
  suppa.py      
```
and this test doesn't fail.